src/main/java/step3/RandomNumberGenerator.java

```diff
@@ -0,0 +1,25 @@
+package step3;
+
+import java.util.Random;
+
+public class RandomNumberGenerator implements NumberGenerator {
+
+    private static final int MAX = 10;
+
+    private final Random random;
+    private final int maxNumber;
+
+    private RandomNumberGenerator(Random random, int maxNumber) {
+        this.random = new Random();
```

Comment:
```suggestion
        this.random = random;
```

`private final int maxNumber; `
리팩토링하다보면 사용하지 않는 코드들이 발생하게 됩니다.
이러한 코드를 지속적으로 관리하는게 복잡도를 줄이는데에 도움이 된다고 생각해요 😄